doc/main.html

```diff
@@ -370,6 +372,10 @@ <h4>Overall structure</h4>
               <td><a>Foreword</a></td>
               <td>0 or 1</td>
             </tr>
+            <tr>
```

Comment:
I think this should be part of the foreword.

Reply:
I can make it a sub-section of of Foreward, then user text would appear just above that if present.

Reply:
Somehow instead of _This document was prepared by Technology Committee xxxx_

Reply:
resolved in https://github.com/SMPTE/html-pub/pull/315/commits/55f44909e5b0da22950138775e1f1b0b8bca39cf